### ISSUE TITLE
Binder design

### ISSUE THREAD

**Issue by maticmarin** [RosettaCommons/protein_generator#13]
Hi, I love your work and the possibilities that offer being superfast. I was wondering if is it possible in binder design to only design specific residues of the binder knowing the structure of the other protein. So basically if I have a structure of the protein and I know the sequence of another protein it binds, even the structure, by masking possible interface residues only on the one protein and conditioning with the other protein structure(knowing the interface residues hotspots) to design only those positions on the first protein that are masked.

**Comment by 0merle0** [comment]
Yes it should definitley be possible to do this using the contigs and inpaint_seq flag, if you are not familiar with these arguments, I'm happy to give some more detail

**Comment by alessandronascimento** [comment]
Hi Protein Generator folks! Great job with this tool!

In line with @maticmarin 's question, I was wondering whether it would be possible to 'score' binders designed in protein generator using ipTM (as in AlphaFold multimer) instead of pLDDT. I understand that using  pLDDT makes a lot of sense when we are designing a binder and we want it to be stable. However, I guess ipTM might be a better score for binders.... Does it make sense?

**Comment by 0merle0** [comment]
There is no head for predicting ipTM, I think the best thing to do is fold designs from the model in AlphaFold multimer and use the ipTM metric to score.
